src/test/java/core/basesyntax/strategy/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseOperationHandlerTest {
+    private static final int expected = 25;
+    private OperationHandler purchaseOperationHandler;
+
+    @BeforeEach
+     void setUp() {
+        purchaseOperationHandler = new PurchaseOperationHandler();
+        Storage.fruitStorage.put("banana", 50);
+        Storage.fruitStorage.put("apple",25);
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.fruitStorage.clear();
+    }
+
+    @Test
+    void apply_Purchase_Ok() {
```

Comment:
```suggestion
    void apply_purchase_ok() {
```
let's use lowercase after underscore everywhere